app/Http/Requests/BookCreateRequest.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+use Carbon\Carbon;
+
+class BookCreateRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'name' => 'required|min:8',
+            'author' => 'required',
+            'category_id' => 'required|numeric',
+            'price' => 'required|numeric|exists:books,price',
+            'donator_id' => 'required|exists:users,employee_code',
```

Comment:
- why do you use exists donator_id in table users?

Reply:
dạ, tức là em nhập vào trường donator mã nhân viên, nếu có tồn tại người đó trong db thì mới xác nhận là người đó hiến sách